packages/core/src/middleware/koa-guard.ts

```diff
@@ -69,7 +69,7 @@ export default function koaGuard<
         params: params?.parse(ctx.params),
       } as Guarded<GuardQueryT, GuardBodyT, GuardParametersT>; // Have to do t His since it's too complicated for TS
     } catch (error: unknown) {
-      throw new RequestError(GuardErrorCode.InvalidInput, error);
+      throw new RequestError('guard.invalid_input', error);
```

Comment:
just want to confirm, the only reason for us giving up enum code type definition is to leverage i18n phrases.

Reply:
this is typed string actually. found it'll be easier to read and you can also have the benefit of code intelligence. 
![image](https://user-images.githubusercontent.com/14722250/127318701-2f160df4-52ed-4d87-b268-ff359b169821.png)
the type def is in `packages/phrases/src/index.ts`